presto-main/src/main/java/com/facebook/presto/execution/QueryPlan.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.execution;
+
+import com.facebook.presto.cost.PlanNodeCost;
+import com.facebook.presto.sql.planner.Plan;
+import com.facebook.presto.sql.planner.plan.PlanNode;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+public class QueryPlan
+{
+    private final Plan plan;
```

Comment:
I created a new wrapper, because cost calculator was not available in LogicalPlanner. Guess we could add it there though. Will push a fixup for that soon.